bridges/snowbridge/pallets/outbound-queue-v2/runtime-api/src/lib.rs

```diff
@@ -0,0 +1,20 @@
+// SPDX-License-Identifier: Apache-2.0
+// SPDX-FileCopyrightText: 2023 Snowfork <hello@snowfork.com>
+
+//! Ethereum Outbound Queue V2 Runtime API
+//!
+//! * `prove_message`: Generate a merkle proof for a committed message
+
+#![cfg_attr(not(feature = "std"), no_std)]
+use frame_support::traits::tokens::Balance as BalanceT;
+use snowbridge_merkle_tree::MerkleProof;
+
+sp_api::decl_runtime_apis! {
+	pub trait OutboundQueueV2Api<Balance> where Balance: BalanceT
+	{
+		/// Generate a merkle proof for a committed message identified by `leaf_index`.
+		/// The merkle root is stored in the block header as a
+		/// `sp_runtime::generic::DigestItem::Other`
```

Comment:
The merkle root is just returned, right? Not set in the digest via just a runtime API call

Reply:
The runtime API is only to generate `MerkleProof`,  while the merkle root is stored in the block header which can be verified by Beefy.

https://github.com/paritytech/polkadot-sdk/blob/c8d33396345237c1864dfc0a9b2172b7dfe7ac8f/bridges/snowbridge/pallets/outbound-queue/src/lib.rs#L283

Reply:
Remove the comment causing confusion.
https://github.com/paritytech/polkadot-sdk/pull/7402/commits/2a5d6807a0489851158a52a42de71e59865f00a4